src/util.js

```diff
@@ -74,18 +74,25 @@ util.getAttrs = function( tagMarkup, settings )
     }
 };
 
-util.getRemote = function( uri, callback, toDataUri )
+function getRemote( uri, settings, callback, toDataUri )
 {
     if( /^\/\//.test( uri ) )
     {
         uri = "https:" + uri;
     }
 
+    var requestOptions = {
+        uri: uri,
+        encoding: toDataUri ? "binary" : ""
+    };
+
+    if( typeof settings.requestTransform === "function" )
+    {
+        requestOptions = settings.requestTransform( requestOptions ) || requestOptions;
```

Comment:
Sounds reasonable. Also Angular has [similar request and response transformations](https://docs.angularjs.org/api/ng/service/$http#transforming-requests-and-responses), which must return the transformed object. As for returned falsey values, what if we cancel the request on `false` and throw on anything else? If I were to forget again to end the function with `return`, I'd prefer to get an informative exception instead of suddenly cancelled requests.